website/docs/r/shared_image.html.markdown

```diff
@@ -68,7 +68,7 @@ The following arguments are supported:
 
 * `eula` - (Optional) The End User Licence Agreement for the Shared Image.
 
-* `generalized` - (Optional) Should the Operating System present in this Shared Image be generalized? Defaults to `true`. Setting this to `false` creates a specialized imagine which is the equivalent of sysprepping in windows. Changing this forces a new resource to be created.
+* `specialized` - (Optional) Should the Operating System present in this Shared Image be specialized? Defaults to `false`. Setting this to `true` creates a specialized imagine which is the equivalent of sysprepping in windows. Changing this forces a new resource to be created.
```

Comment:
a Generalized image is one where SysPrep has been applied - Specialised is one where the machine data (including the UUID) is reused - which can cause serious issues (I've personally seen production incidents/data corruption from where sysprep wasn't run) - so we should probably have a warning recommending to use a Generalized image where possible

```suggestion
* `specialized` - (Optional) Specifies that the Operating System used inside this Image has not been Generalized (for example, `sysprep` on Windows has not been run). Defaults to `false`. Changing this forces a new resource to be created.

!> **Note:** It's recommended to Generalize images where possible - Specialized Images reuse the same UUID internally within each Virtual Machine, which can have unintended side-effects.
```

Reply:
I understand. I will soon make another PR to update this doc entry